### PR TITLE
Refactor to have an actual picker event with event interface rehaul

### LINKED ISSUE #53: (Re)make the event interface
**Is your feature request related to a problem? Please describe.**

Make an event interface (that is just pure data).

**Describe the solution you'd like**
No functions pls.

**Additional context**
no.

### LINKED ISSUE #55: Faculty picking should be an event and not one-off
**Describe the part of the code base that needs to be cleaned up**
Faculty picking should be an event and not one-off functions.

**Provide suggestions on how we can clean this up**
Just make a new event how hard is that??????

### PR DESCRIPTION
**Pull request by cheukyin699**
# :ship: Pull Request

## :question: Purpose

**Give a brief description of your changes:** Remove the sh!tty event and add a faculty picker event for everyone to see.

## :tickets: Ticket(s)

Closes #53
Closes #55

### CODE REVIEW

File: src/IEvent.ts [hunk 1]
```diff
@@ -1,13 +1,23 @@
-// import Player from "./Player";
+export interface IEvent {
```
Comment (eloisepeng) [inline src/IEvent.ts]: shouldn't we change the name for this file to be more descriptive?

File: src/Player.tsx [hunk 2]
```diff
@@ -1,7 +1,5 @@
-import IEvent from "./IEvent";
-import ShittyEvent from "./ShittyEvent";
-
-const DEFAULT_SLEEP = 100;
+import { IEvent }from "./IEvent";
+import PickFaculty from "./events/PickFaculty";
```
Comment (eloisepeng) [inline src/Player.tsx]: FacultyPicker maybe better?

File: src/Player.tsx [hunk 3]
```diff
@@ -1,7 +1,5 @@
-import IEvent from "./IEvent";
-import ShittyEvent from "./ShittyEvent";
-
-const DEFAULT_SLEEP = 100;
+import { IEvent }from "./IEvent";
```
Comment (eloisepeng) [inline src/Player.tsx]: if we are not using react, this should be .ts instead of .tsx

File: src/IEvent.ts (hunk 1 above)
Comment (cheukyin699) [reply src/IEvent.ts]: Would just `src/Event.ts` be good?

Comment (eloisepeng) [reply src/IEvent.ts]: sure

**Pull request merged**